pytorch_pfn_extras/engine.py

```diff
@@ -138,33 +140,40 @@ def create_trainer(
             If `None` is given, the evaluation is skipped.
         device (str or torch.device):
             Device name used for selecting a corresponding runtime class.
-        options (dict):
-            Option that is set to a logic object. When using the default logic
-            class, See the documentation of `ppe.handler.Logic` for details.
         logic (logic object):
             A logic object. If `None` is given, an logic object is instantiated
             from the default logic class.
         handler_class (handler class):
             A handler class that instantiates a handler object. If `None` is
             given, `ppe.handler.Handler` is used as a default handler class.
+        handler_options (dict):
+            Options that are set to the handler object. See the documentation
+            of `ppe.handler.Handler` for details.
+        logic_options (dict):
+            Options that arte set to the logic object. When using the default
+            logic class, See the documentation of `ppe.handler.Logic`
+            for details.
+        handler_options (dict):
+            Options that are set to the runtime object. See the documentation
+            of `ppe.handler.Handler` for details.
```

Comment:
```suggestion
        handler_options:
            Options that are set to the handler object. See the documentation
            of `ppe.handler.Handler` for details.
        logic_options:
            Options that are set to the logic object. When using the default
            logic class, See the documentation of `ppe.handler.Logic`
            for details.
        runtime_options:
            Options that are set to the runtime object. See the documentation
            of `ppe.handler.PyTorchRuntime` for details.
```

Reply:
All the above params in the doc string have their type specified, should we remove all?

Reply:
Updated suggestion, let's keep it as-is and discuss in #314 about that :smile: